pkg/scheduler/framework/plugins/noderesources/test_util.go

```diff
@@ -22,6 +22,83 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type containerResource struct {
+	cpu    int
+	memory int
+}
+
+func makePod(node string, milliCPU, memory int64) *v1.Pod {
+	return &v1.Pod{
```

Comment:
this can be achieved by `st.MakePod().Node().Req()`

Reply:
great! thanks for pointing this out.